### PR TITLE
created users table and inserted dummy data

### PR DESCRIPTION
**Pull request by mjmandelah07**
Please inspect

### CODE REVIEW

File: db/queries/createUsersAccount.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+const dbConfig = {
+  user: process.env.DB_USER,
+  host: "localhost",
+  database: process.env.DB_DATABASE,
+  password: process.env.DB_PASSWORD,
+  port: 5432,
+};
+
+const client = new Client(dbConfig);
```
Comment (kelvinator07) [inline db/queries/createUsersAccount.js]: This Should Be In **db.js** file, Then Import From Anywhere If You Want To Communicate With The DB

File: db/queries/createUsersAccount.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+    // Insert dummy user data
+    await client.query(insertUserDataQuery, values);
+    console.log('Dummy user data inserted');
+  } catch (error) {
+    console.error('Error:', error);
+  } finally {
+    // end database connection
+    await client.end();
+  }
```
Comment (kelvinator07) [inline db/queries/createUsersAccount.js]: Dont Close The Connection, Leave It Open So You Can Continue To Communicate With The DB

File: db/queries/createUsersAccount.js (hunk 1 above)
Comment (mjmandelah07) [reply db/queries/createUsersAccount.js]: i tried it, but it is giving me that error and im not able to solve it with all the solution online

File: package.json [hunk 3]
```diff
@@ -10,10 +10,12 @@
   "author": "mojisola aramide",
   "license": "ISC",
   "dependencies": {
+    "bcrypt": "^5.1.1",
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
     "nodemon": "^3.0.1",
-    "pg": "^8.11.2"
+    "pg": "^8.11.2",
+    "pg-promise": "^11.5.3"
```
Comment (kelvinator07) [inline package.json:19]: Why Did You Add Another Dependency?

File: db/queries/createUsersAccount.js (hunk 2 above)
Comment (mjmandelah07) [reply db/queries/createUsersAccount.js]: okay

File: package.json [hunk 4]
```diff
@@ -10,10 +10,12 @@
   "author": "mojisola aramide",
   "license": "ISC",
   "dependencies": {
+    "bcrypt": "^5.1.1",
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
     "nodemon": "^3.0.1",
```
Comment (kelvinator07) [inline package.json:17]: Nodemon Is a Dev Dependency

File: package.json (hunk 3 above)
Comment (mjmandelah07) [reply package.json:19]: mistake

File: package.json (hunk 4 above)
Comment (mjmandelah07) [reply package.json:17]: I dont understand this, i use it to start index.js

Comment (kelvinator07) [reply package.json:17]: Yes But Its For Development Purpose

**Review by kelvinator07 — APPROVED** [review]
